src/redux/config.js

```diff
@@ -64,8 +64,8 @@ const HF_DESC_URL = 'https://support.bitfinex.com/hc/en-us/articles/900000096823
 const DISCUSSION_BOARD_URL = 'https://bit.ly/42p9YiV'
 const DISCORD_URL = 'https://bit.ly/400Xj49'
 
-const CHART_URL = isElectronApp ? 'https://bitfinexcom.github.io/bfx-hf-tradingview/' : process.env.REACT_APP_CHART_URL
-// const CHART_URL = 'http://localhost:3001/bfx-hf-tradingview/'
+// const CHART_URL = isElectronApp ? 'https://bitfinexcom.github.io/bfx-hf-tradingview/' : process.env.REACT_APP_CHART_URL
+const CHART_URL = 'http://localhost:3001/bfx-hf-tradingview/'
```

Comment:
is this for local testing and should be removed now ?

Reply:
for local testing for BE devs and Ivan. should not be merge before resolving some issues 
https://app.asana.com/0/1201848935859401/1204583390653630
https://app.asana.com/0/1201848935859401/1204589517355550

Reply:
moving to Draft until dependencies points to bfx package versions